lib/writer/error-types.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
     if (process.platform === 'darwin') {
       return error.code === 'ENXIO' || error.code === 'EBUSY'
     } else if (process.platform === 'linux') {
-      return error.code === 'EIO'
+      return error.code === 'EIO' || error.code === 'EBUSY'
```

Comment:
If this **is** what we want to do, then the comment on line 27 needs updating ;-)

Reply:
bump ;0

Reply:
Fixed